drivers/backup/portworx/portworx.go

```diff
@@ -1535,6 +1617,166 @@ func (p *portworx) DeleteBackupSchedulePolicy(orgID string, policyList []string)
 	return nil
 }
 
+// CreateRuleForBackupWithMultipleApplications creates backup rule for multiple application in one rule
+func (p *portworx) CreateRuleForBackupWithMultipleApplications(orgID string, appList []string, ctx context.Context, appParameters ...map[string]backup.AppRule) (string, string, error) {
+	var (
+		preUid             string
+		preRuleName        string
+		postRuleName       string
+		postUid            string
+		preActionValue     []string
+		preContainer       []string
+		postActionValue    []string
+		postContainer      []string
+		postBackground     []bool
+		postRunInSinglePod []bool
+		preBackground      []bool
+		preRunInSinglePod  []bool
+		preRulesInfo       api.RulesInfo
+		postRulesInfo      api.RulesInfo
+		prePodSelector     []map[string]string
+		postPodSelector    []map[string]string
+	)
+	if len(appParameters) == 0 {
+		appParameters = append(appParameters, AppRuleMaster)
+	}
```

Comment:
Can we do

```
if len(appParameters) == 0 {
		appParameter = AppRuleMaster
	} else {
                appParameter = appParameters[0]
}
```
Then we can just keep using `appParameter` instead of `appParameters[0]` everywhere

Reply:
done